com.hma.qlog.yml

```
@@ -0,0 +1,43 @@
+app-id: com.hma.qlog
```

```suggestion
app-id: io.github.7h0ma5.QLog
```

Similar change is needed for desktop, appdata, etc.

---

ok

---

In the name 7h0ma5 of the application is not valid I imagine that 7h0ma5 refers to Thomas I will have to give him that name

---

error: 'io.github.7h0ma5.QLog' is not a valid application name: Name segment can't start with 7